sass/pages/check-mail/_check_mail.scss

```diff
@@ -0,0 +1,56 @@
+.check-body{
+	background-color: #F7F7F7;
+
+	.check-section{
+		background-color: $white;
+		text-align: center;
+
+		.check-div{
+			width: 30%;
+			@include center(true,true);
```

Comment:
Do not use this mixin unless it's truly necessary, find another approach for this.